src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -235,6 +235,24 @@
      * modifiers, type, the identifier name, and an optional
      * assignment statement.
      *
+     * <p>For example:</p>
+     * <pre>
+     * private int PI = 3.14;
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * |--VARIABLE_DEF -&gt; VARIABLE_DEF
```

Comment:
please remove leading `|`  it goes to nowhere and not required

Reply:
`|` waas added on advice of previous reviewer @timurt

Reply:
Also changed literal `private` to `final`